### ISSUE TITLE
Consistent ordering only works with --num-executors 1

### ISSUE THREAD

**Issue by Jeinhaus** [transcend-io/terragrunt-atlantis-config#145]
Hi,

When generating the `atlantis.yml` using v1.6.0 the order in the generated file changes with every run.
I'm using this call:
```
terragrunt-atlantis-config generate --ignore-parent-terragrunt --autoplan --workflow terragrunt --parallel=false --output ./atlantis.yaml
```

The order is only consistent when appending `--num-executors 1`.
Let me know if you need any more details.

**Comment by dmattia** [comment]
Interesting, https://github.com/transcend-io/terragrunt-atlantis-config/pull/141 was meant to make the ordering consistent.

I'm curious, how large is your atlantis.yaml output, or how many projects do you have? These sorts of bugs are always tricky to test. We use `--num-executors 15` on our tests, and they run consistently, but few tests have more than 20 projects that get built

**Comment by Jeinhaus** [comment]
This specific repo contains 116 projects. I just tried it in another repo with the same basic structure but only 6 projects.
The ordering in either of them is only consistent if I configure 1 executor.

**Comment by dmattia** [comment]
Could you please create a PR with the structure under the https://github.com/transcend-io/terragrunt-atlantis-config/tree/master/test_examples folder of this repo? These are all of the current structures that we test, and I can gladly go the rest of the way and write tests that run against that structure multiple times each test run

**Comment by fbad** [comment]
Yes, actually #141 does not guarantee consistent ordering of projects, it just fixes the tests, I did not think it might be a problem.
I will open a PR with a fix.

**Comment by dmattia** [comment]
This should be fixed in the v1.7.0 release, thanks @fbad!

https://github.com/transcend-io/terragrunt-atlantis-config/releases/tag/v1.7.0
